examples/mnist/train.cpp

```diff
@@ -45,14 +45,22 @@ void construct_net(network<sequential>& nn) {
 #undef O
 #undef X
 
+    core::backend_t backend_type = core::backend_t::tiny_cnn;
+    backend_type = core::backend_t::avx;
+
```

Comment:
The reason is standard backend type is `core::backend_t::tiny_cnn`, and avx_backend is still experimental. Another reason is I was still debugging the code. I guess I should remove secondary assign code from the PR.

> `backend_type = core::backend_t::avx;`